src/styles/blocks/calendar.scss

```
@@ -0,0 +1,32 @@
+.calendar {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: flex-start;
+  align-items: center;
+
+  padding: 150px 10px;
+  gap: 1px;
+
+  max-width: $max-width;
+
+  &__cell {
+    width: $cell-width;
+    height: $cell-width;
+
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    border: 1px solid;
+    background: #eee;
+
+    transition: color 0.5s, transform 0.5s;
```

Create a variable for transition duration
```suggestion
    transition: color $transitionDuration, transform $transitionDuration;
```